blocks/columns/columns.css

```diff
@@ -65,7 +65,7 @@
 
 .bg-primary-blue-90 .columns.columns-transform-teaser p:nth-child(3) a {
   background-color: var(--white);
-  color: var(--accent-blue-primary-90);
+  color: var(--primary-blue-70);
```

Comment:
Any reason for reverting to old color variable?

Reply:
Color code for --accent-blue-primary-90 variable is '#012740'  whereas color code in figma is '#004370' so changed the variable @thomsebastin

Reply:
But isn't there a new variable equivalent for that colour? Can you please check?

Reply:
changed into accent-blue-primary-70 @thomsebastin.
pls check